hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -415,11 +422,34 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
 
         # TODO: status.sideBackWindowHeat
         # TODO: status.sideMirrorHeat
-        # TODO: status.seatHeaterVentState.flSeatHeatState
-        # TODO: status.seatHeaterVentState.frSeatHeatState
-        # TODO: status.seatHeaterVentState.rlSeatHeatState
-        # TODO: status.seatHeaterVentState.rrSeatHeatState
-        # TODO: status.doorLock
+
+        flSeatHeatState = get_child_value(state, "Cabin.Seat.Row1.Driver.Climate.State")
+        if flSeatHeatState in [0, 2]:
+            vehicle.front_left_seat_status = False
+        elif flSeatHeatState == 1:
+            vehicle.front_left_seat_status = True
+
+        frSeatHeatState = get_child_value(
+            state, "Cabin.Seat.Row1.Passenger.Climate.State"
+        )
+        if frSeatHeatState in [0, 2]:
```

Comment:
This appears to be status like all other regions not is_on.  As such should be using the constants. 

https://github.com/Bubo08/hyundai_kia_connect_api/blob/7bf5c6444cd7c0049b065df98685d8280d7f31a6/hyundai_kia_connect_api/const.py#L38-L49

Other states most likely exist for cars as per the above.

Reply:
Good point. I did not realize that there were status in the code. I fixed the above. 
I think I will have to check some other attributes.

Reply:
Sounds good.  What I have found in the past is anything that's a number in the api could be more than true false.  We just don't know it yet. 

We hit that on seats.  For a long time it was 0 and 1. Until we started seeing other options for more advanced cars.  We can cross that bridge when we come to it though.

Reply:
The code is now more readable. So this is good change in my opinion :-)